microsoft-edge/devtools-guide-chromium/memory-problems/index.md

```diff
@@ -250,7 +250,27 @@ DevTools shows you a breakdown of memory allocation by function.  The default vi
 
 ![Allocation sampling.](../media/memory-problems-glitch-example-05-memory-allocation-sampling-heavy-bottom-up.msft.png)
 
+## Use settings for Allocation sampling to reduce garbage
 
+By default, the **Allocation sampling** profiling type only reports allocations that are still alive at the end of the recording session.  Objects that are generated, garbage collected (GC'd), and then disappear aren't tracked by either **Allocation sampling** or **Allocation instrumentation on timeline**.
+
+You can configure the **Allocation sampling** profiling type to track objects that are discarded by GC with settings.
+
+1. Click the **Allocation sampling** radio button.
+
+1. Click the **Include objects discarded by major GC** and **Include objects discarded by minor GC** settings.
+
+    ![Allocation sampling GC settings.](../media/memory-problems-memory-allocation-sampling-gc-settings.png)
```

Comment:
This image needs a red box around the two settings in question. Otherwise one has to scan the entire image to find them.

Reply:
@mikehoffms or @JasonAndrewWriter, can you help me with this PR by drawing red boxes on the images? For the first one, draw red boxes around the blue line when it steps down. For the second one, draw a red box around the settings under **Allocation sampling**

Reply:
will chat w/ Jason re: committing such a change.  

gh rendered:
https://github.com/MicrosoftDocs/edge-developer/blob/user/zoghadya/allocation-sampling-gc-settings/microsoft-edge/devtools-guide-chromium/memory-problems/index.md#use-settings-for-allocation-sampling-to-reduce-garbage

Reply:
> @mikehoffms or @JasonAndrewWriter, can you help me with this PR by drawing red boxes on the images? For the first one, draw red boxes around the blue line when it steps down. For the second one, draw a red box around the settings under **Allocation sampling**

I have it set for first thing in the morning.  Just wanted to make sure i could find the thread

Reply:
I just uploaded these going to ask @zoherghadyali to take a look